### DISCUSSION TITLE
将Django模版改为Jinja2来提升速度

### DISCUSSION THREAD

**Discussion by Fgaoxing** [Qexo/Qexo#420] [Ideas]
https://docs.jinkan.org/docs/jinja2/switching.html#django

**Comment by Fgaoxing** [comment]
![image](https://github.com/Qexo/Qexo/assets/92655031/99367b53-0c20-4e1e-8cf8-f4c219041a55)

**Comment by am-abudu** [comment]
我觉得没有人会在意后台页面的加载速度

**Comment by Fgaoxing** [comment]
但是需要避免超时问题，而几乎通用，只要简单更改即可



在 2023-07-29 00:54:59，"Abudu" ***@***.***> 写道：

我觉得没有人会在意后台页面的加载速度
